app/views/model_view.py

```diff
@@ -62,12 +61,12 @@ def put(self, request):
             donor.want_receipt = "wantReceipt" in request.PUT
             donor.telephone_number = request.PUT["telephoneNumber"]
             donor.mobile_number = request.PUT["mobileNumber"]
-            donor.address_line = request.PUT["addressLine"]
+            donor.address_line_one = request.PUT["addressLineOne"]
```

Comment:
request wouldn't have addressLineOne haha https://github.com/CodeTheChangeUBC/reBOOT/blob/3aff13c3ba6cf9fadee48ce2bd0c7366c304f2e3/app/templates/app/form.html#L78

Reply:
Is that form also deprecated?